filebeat/docs/inputs/input-container.asciidoc

```diff
@@ -3,6 +3,12 @@
 [id="{beatname_lc}-input-{type}"]
 === Container input
 
+deprecated:[7.16.0]
```

Comment:
This is a retroactive deprecation isn't it? There's no deprecation warning right now https://www.elastic.co/guide/en/beats/filebeat/current/filebeat-input-container.html

We are the only people who know this is the log input underneath, we should explain we consider this deprecated because it is just the log input underneath and can be replaced by filestream directly.